storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/RemoteLogMetadataFormatterTest.java

```diff
@@ -35,28 +34,29 @@
 import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.apache.kafka.server.log.remote.storage.RemoteLogSegmentState.COPY_SEGMENT_STARTED;
 
 public class RemoteLogMetadataFormatterTest {
-    private static final Uuid TOPIC_ID = Uuid.randomUuid();
```

Comment:
Those changes are a kind of "code style". I guess there is no strict rules in our code base. However, I prefer to reduce the code changes if we don't have strong reason.

Reply:
It's kind of grey zone to me. I'm not so sure why make it non-static.

Reply:
restored the static variables.